security/abac/role/src/main/java/io/helidon/security/abac/role/RoleValidator.java

```diff
@@ -329,6 +372,28 @@ private Builder addServiceRole(String role) {
                 return this;
             }
 
+            /**
+             * Sets if all access should be permitted.
+             *
+             * @param permitAll if access should be permitted for all
+             * @return updated builder instance
+             */
+            private Builder permitAll(boolean permitAll) {
+                this.permitAll = permitAll;
+                return this;
+            }
+
+            /**
+             * Sets if all access should be denied.
+             *
+             * @param denyAll if access should be denied for all
+             * @return updated builder instance
+             */
+            private Builder denyAll(boolean denyAll) {
+                this.denyAll = denyAll;
+                return this;
+            }
+
```

Comment:
Missing possibility to add permitAll and denyAll from configuration - method config(Config)

Reply:
Support for handling permitAll and denyAll from config implemented. (Tests included)